cpp/src/arrow/flight/sql/client.cc

```diff
@@ -31,6 +31,7 @@
 #include "arrow/result.h"
 #include "arrow/util/logging.h"
 
+namespace pb = arrow::flight::protocol;
```

Comment:
Considered doing this as `flight_pb` however this is more consistent with usage elsewhere and could eliminate uses of `flight_sql_pb` for better uniformity.